help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,213 @@
+| Export frequency | **[!UICONTROL Streaming]** | <ul><li>Streaming destinations are "always on" API-based connections. As soon as a profile is updated in Experience Platform based on segment evaluation, the connector sends the update downstream to the destination platform. Read more about [streaming destinations](/help/destinations/destination-types.md#streaming-destinations).</li></ul>|
+
+{style="table-layout:auto"}
+
+## Connect to the destination {#connect}
+
+> [!IMPORTANT]
+> 
+> To connect to the destination, you need the **[!UICONTROL Manage Destinations]** [access control permission](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+To connect to this destination, follow the steps described in the [destination configuration tutorial](../../ui/connect-destination.md). In the configure destination workflow, fill in the fields listed in the two sections below.
+
+![Catalog](../../assets/catalog/crm/microsoft-dynamics-365/catalog.png)
+
+### Authenticate to destination {#authenticate}
+
+To authenticate to the destination, select **[!UICONTROL Connect to destination]**.
+
+![Sample screenshot showing how to authenticate to Dynamics 365](../../assets/catalog/crm/microsoft-dynamics-365/authenticate-destination.png)
+
+You will be shown the [!DNL Dynamics 365] login page. Select your account if you are already signed-in or login with your credentials.
+
+![Sample screenshot showing how to authenticate to Dynamics 365](../../assets/catalog/crm/microsoft-dynamics-365/authenticate-destination-login.png)
+
+*  **[!UICONTROL Password]**: Your [!DNL Dynamics 365] account password.
+*  **[!UICONTROL Username]**: Your [!DNL Dynamics 365] account username.
+
+If the details provided are valid, the UI displays a **Connected** status with a green check mark, you can then proceed to the next step.
+
+### Fill in destination details {#destination-details}
+
+To configure details for the destination, fill in the required and optional fields below. An asterisk next to a field in the UI indicates that the field is required.
+![Sample screenshot showing how to fill in details for Dynamics 365 destination](../../assets/catalog/crm/microsoft-dynamics-365/destination-details.png)
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Environment URL]**: Your [!DNL Dynamics 365] Environment URL.
```

Comment:
Can we add a link to the Dynamics 365 docs where users can see how to obtain the Environment URL?